css/Timer.css

```diff
@@ -0,0 +1,40 @@
+body{
+    background-color: #030334;
+}
+/*fourth take */
+
+*{
+    box-sizing: border-box;
+}
+
+.watch{
+  display: flex;
+    justify-content: center;
+    align-items: center;
+    background-color: lightblue;
+    width: 100%;
+    max-width: 360px;
+    margin: 4rem auto;
+    padding: 1rem 2rem;
+    border-radius: 1rem;
+    flex-wrap: wrap;
+}
+
+.display{
+    margin: 5px;
+    font-size: 4rem;
+}
+.buttons{
+    display: flex;
+
+
+}
+#start{
+    margin: 5px;
+}
```

Comment:
Any reason you used class selector for some and ID selectors for others? (for example .button vs #start)

Reply:
I thought maybe I would have some other buttons later on and I would possibly like to keep the css same for them. but start, stop and reset buttons are very specific and they will not be used anywhere else probably.